eng/pipelines/jobs/build.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       ${{ if eq(parameters.architecture, 'arm64') }}:
         container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-cross-arm64
       ${{ else }}:
-        container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
+        container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04
```

Comment:
Does 2.27 glibc have any incompatibilities with 2.23?

Reply:
In the future, we're likely to split test phase into its own set of jobs where we run all of the tests on different distros to validate the backward compatibility of newer distros with old glibc reference in the profiler.